<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TheGame</class>
 <widget class="QMainWindow" name="TheGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TheGame</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1200</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <widget class="QLabel" name="player">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>60</width>
      <height>87</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="files.qrc">:/player/Imgs/player.png</pixmap>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="forest">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="files.qrc">:/images/Imgs/Forest.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1200</width>
      <height>600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(246, 244, 243)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>4</number>
    </property>
    <widget class="QLabel" name="AmountOfGold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255,215,0)</string>
     </property>
     <property name="text">
      <string>GOLD  : 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="AmountOfStone">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(105,105,105)</string>
     </property>
     <property name="text">
      <string>STONE : 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="AmountOfWood">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:	rgb(139,69,19)</string>
     </property>
     <property name="text">
      <string>WOOD  : 0</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="quarry">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>221</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="files.qrc">:/images/Imgs/Quarry.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="buildingSite">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>366</y>
      <width>181</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="files.qrc">:/images/Imgs/buildingsite.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load game</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save game</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New game</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="files.qrc"/>
 </resources>
 <connections/>
</ui>
